packages/protocol/script/AuthorizeRelayer.s.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+//  _____     _ _         _         _
+// |_   _|_ _(_) |_____  | |   __ _| |__ ___
+//   | |/ _` | | / / _ \ | |__/ _` | '_ (_-<
+//   |_|\__,_|_|_\_\___/ |____\__,_|_.__/__/
+//
+//   Email: security@taiko.xyz
+//   Website: https://taiko.xyz
+//   GitHub: https://github.com/taikoxyz
+//   Discord: https://discord.gg/taikoxyz
+//   Twitter: https://twitter.com/taikoxyz
+//   Blog: https://mirror.xyz/labs.taiko.eth
+//   Youtube: https://www.youtube.com/@taikoxyz
+
+pragma solidity 0.8.24;
+
+import "../test/DeployCapability.sol";
+import "../contracts/L1/gov/TaikoTimelockController.sol";
+import "../contracts/signal/SignalService.sol";
+
+contract AuthorizeRelayer is DeployCapability {
```

Comment:
lets rename it to AuthorizeTaikoForMultihop

Reply:
Renamed